profile/README.md

```diff
@@ -12,9 +12,6 @@ High-Performance Computing (HPC), Internet of Things (IoT), and Machine Learning
 
 It is easy to get started with HSE.
 
-* [Read the documentation to learn more about HSE](https://hse-project.github.io/)
-* [Build HSE for your Linux distribution](https://github.com/hse-project/hse)
-* [View blog posts, white papers, and other information on HSE](https://micron.com/hse)
-
-You can also learn more about
-[contributing](https://github.com/hse-project/hse/blob/master/CONTRIBUTING.md) to HSE.
+* Read the [documentation](https://hse-project.github.io/) to learn more about HSE
+* [Build](https://github.com/hse-project/hse) HSE for your Linux distribution
+* View blog posts, white papers, and other information on our [home site](https://micron.com/hse)
```

Comment:
Does homepage make more sense?

Reply:
I don't know if it makes more sense, but googling it gets more hits so will change.